xwiki-platform-core/xwiki-platform-messagestream/xwiki-platform-messagestream-api/pom.xml

```diff
@@ -31,7 +31,7 @@
   <name>XWiki Platform - Message Stream - API</name>
   <description>API to publish and display user status messages in different streams</description>
   <properties>
-    <xwiki.jacoco.instructionRatio>0.69</xwiki.jacoco.instructionRatio>
+    <xwiki.jacoco.instructionRatio>0.70</xwiki.jacoco.instructionRatio>
```

Comment:
did you run it on 11.10.x? We have surprises sometimes with the coverage on different branches...

Reply:
Good question. I've run the test locally with success.